releasenotes/releasenotes.rst

```diff
@@ -24,11 +24,56 @@ Public Endpoint Announcement 8 July 2021
 
 The default FHIR version of the `public Firely Server endpoint <https://server.fire.ly/>`_ is now R4.
 
+
+.. _vonk_releasenotes_460:
+
+Release 4.6.0
```

Comment:
As requested by Frank on Slack:
```suggestion
Release 4.6.0, Nov 11th, 2021
```

Reply:
I have put `Release 4.6.0, Nov 18th, 2021` into the documentation assuming that 11th is a typo